requirements/edx/base.txt

```diff
@@ -639,7 +639,7 @@ httplib2==0.22.0
     # via
     #   google-api-python-client
     #   google-auth-httplib2
-icalendar==5.0.13
+icalendar==6.0.1
```

Comment:
Version `6.0.0` removed support for `Python 3.7` and started using `zoneinfo` internally. [[CHANGELOG](https://icalendar.readthedocs.io/en/latest/changelog.html#id2)]